ui/components/ui/account-list/account-list.js

```diff
@@ -20,6 +20,13 @@ const AccountList = ({
   handleAccountClick,
 }) => {
   const t = useI18nContext();
+  const selectedAccountScrollRef = useRef(null);
+
+  useLayoutEffect(() => {
+    if (selectedAccountScrollRef.current) {
+      selectedAccountScrollRef.current.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Not sure if we have a style preference but we could avoid the `if` and just do:

```
selectedAccountScrollRef.current?.scrollIntoView({ behavior: 'smooth' });
```

Reply:
Updated!